streaming/streaming.go

```diff
@@ -253,33 +308,41 @@ func getPostgreSQLSourceStreamConfig(properties string) (*datastreampb.SourceCon
 	if !replicationSlotExists || !publicationExists {
 		return nil, fmt.Errorf("replication slot or publication not specified")
 	}
+	if datastreamCfg.MaxConcurrentBackfillTasks != "" {
+		intVal, err := strconv.ParseInt(datastreamCfg.MaxConcurrentBackfillTasks, 10, 64)
+		if err != nil {
+			return &datastreampb.SourceConfig_PostgresqlSourceConfig{}, fmt.Errorf("could not parse maxConcurrentBackfillTasks parameter %s, please provide a positive integer as input", datastreamCfg.MaxConcurrentBackfillTasks)
+		}
+		maxBackfillTasks = int32(intVal)
+		if maxBackfillTasks < MIN_DATASTREAM_TASK_LIMIT || maxBackfillTasks > MAX_DATASTREAM_TASK_LIMIT {
+			return &datastreampb.SourceConfig_PostgresqlSourceConfig{}, fmt.Errorf("maxConcurrentBackfillTasks should lie in the range [%d, %d]", MIN_DATASTREAM_TASK_LIMIT, MAX_DATASTREAM_TASK_LIMIT)
+		}
+	}
 	postgresSrcCfg := &datastreampb.PostgresqlSourceConfig{
 		ExcludeObjects:             &datastreampb.PostgresqlRdbms{PostgresqlSchemas: excludeObjects},
 		ReplicationSlot:            replicationSlot,
 		Publication:                publication,
-		MaxConcurrentBackfillTasks: 50,
+		MaxConcurrentBackfillTasks: maxBackfillTasks,
```

Comment:
PG does not support the CDC configuration?

Reply:
Not yet, it only allows backfill customisation currently.